apps/bajour/src/components/website-builder-overwrites/blocks/teaser-overwrite.style.tsx

```diff
@@ -37,11 +36,13 @@ export const TeaserImgStyled = styled(Image)`
   aspect-ratio: 3/2;
 `
 
-export const TeaserContentStyled = styled(TeaserContent)``
+export const TeaserContentStyled = styled(TeaserInnerWrapper)`
+  grid-column: initial;
+`
 
-export const TeaserPreTitleStyled = styled(TeaserPreTitle)``
+export const TeaserPreTitleStyled = styled('span')``
 
-export const TeaserTitlesStyled = styled(H3)`
+export const TeaserTitlesStyled = styled('h1')`
```

Comment:
do you think that h1 is the best choice? shouldn't there be one h1 per page ideally?